updates/pgbouncer_fdw--0.5--1.0.0.sql

```diff
@@ -0,0 +1,1442 @@
+DROP VIEW @extschema@.pgbouncer_users;
+
+DROP FUNCTION @extschema@.pgbouncer_command_disable(text);
+DROP FUNCTION @extschema@.pgbouncer_command_enable(text);
+DROP FUNCTION @extschema@.pgbouncer_command_kill(text);
+DROP FUNCTION @extschema@.pgbouncer_command_pause(text);
+DROP FUNCTION @extschema@.pgbouncer_command_reconnect(text);
+DROP FUNCTION @extschema@.pgbouncer_command_reload();
+DROP FUNCTION @extschema@.pgbouncer_command_resume(text);
+DROP FUNCTION @extschema@.pgbouncer_command_set(text, text);
+DROP FUNCTION @extschema@.pgbouncer_command_shutdown();
+DROP FUNCTION @extschema@.pgbouncer_command_suspend();
+DROP FUNCTION @extschema@.pgbouncer_command_wait_close(text);
+
+/*
+ * pgbouncer_fdw_targets 
+ */
+CREATE TABLE @extschema@.pgbouncer_fdw_targets (
+    target_host text NOT NULL
+    , active boolean NOT NULL DEFAULT true
+    , CONSTRAINT pgbouncer_fdw_targets_pk PRIMARY KEY (target_host) );
+CREATE INDEX pgbouncer_fdw_targets_active_idx ON pgbouncer_fdw_targets (active);
+SELECT pg_catalog.pg_extension_config_dump('pgbouncer_fdw_targets', '');
+
+INSERT INTO @extschema@.pgbouncer_fdw_targets ( target_host ) VALUES ('pgbouncer');
+
+
+/*
+ * pgbouncer_version_func
+ */
+CREATE FUNCTION  @extschema@.pgbouncer_version_func(p_target_host text DEFAULT NULL) RETURNS TABLE
+(
+    pgbouncer_target_host text
+    , version_major int
+    , version_minor int
+    , version_patch int
+)
+LANGUAGE plpgsql
+AS $$
+DECLARE
+    ex_context                      text;
+    ex_detail                       text;
+    ex_hint                         text;
+    ex_message                      text;
+    v_row   record;
+    v_sql   text;
+BEGIN
+
+v_sql := 'SELECT target_host FROM @extschema@.pgbouncer_fdw_targets WHERE active';
+IF p_target_host IS NOT NULL THEN
+    v_sql := v_sql || format(' AND target_host = %L', p_target_host);
+END IF;
+
+FOR v_row IN EXECUTE v_sql
+LOOP BEGIN
+    RETURN QUERY SELECT 
+        v_row.target_host AS pgbouncer_target_host
+        , split_part(substring(version from '\d.+'), '.', 1)::int AS version_major
+        , split_part(substring(version from '\d.+'), '.', 2)::int AS version_minor
+        , split_part(substring(version from '\d.+'), '.', 3)::int AS version_patch
+    FROM dblink(v_row.target_host, 'show version') AS x
+    (   
+        version text
+    );
+    EXCEPTION
+        WHEN connection_exception THEN
+            GET STACKED DIAGNOSTICS ex_message = MESSAGE_TEXT,
+                                    ex_context = PG_EXCEPTION_CONTEXT,
+                                    ex_detail = PG_EXCEPTION_DETAIL,
+                                    ex_hint = PG_EXCEPTION_HINT;
+            RAISE WARNING 'pgbouncer_fdw: Unable to establish connection to pgBouncer target host: %. Continuing to additional hosts.
+ORIGINAL ERROR: %
+CONTEXT: %
+DETAIL: %
+HINT: %', v_row.target_host, ex_message, ex_context, ex_detail, ex_hint;
+END;
+END LOOP;
+END
+$$;
+
+CREATE VIEW @extschema@.pgbouncer_version AS
```

Comment:
Missing the `pgbouncer_target_host` field?

Reply:
Fixed